xds/src/test/java/io/grpc/xds/XdsClientImplTestForListener.java

```diff
@@ -109,7 +109,7 @@
       "type.googleapis.com/"
           + "envoy.config.filter.network.http_connection_manager.v2.HttpConnectionManager";
 
-  private static final Node NODE = Node.getDefaultInstance();
+  private static final Node NODE = Node.newBuilder().build();
```

Comment:
nit: (this is not for your change) This `Node` is not a constant, so it should not named in upper-case.

Reply:
Reverted.

Reply:
Why is it not constant?

Reply:
Because it gets modified based on the port passed to it (adding the listening_address etc). I admit I left it as `NODE` even though it was no longer a constant.

Reply:
> Because it gets modified based on the port passed to it (adding the listening_address etc). 

After the change, `NODE` is not modified.

Reply:
I'd say the Node here is unmodified, but its usage is not as a _constant_ (compare to how the integer `PORT` is used). So I would not treat it as a constant.

Alright, this is the test code, not a big issue. I am not gonna be too picky for this small nit.